### ISSUE TITLE
Document correct ES6 module usage to avoid "SyntaxError: import not found: default"

### ISSUE THREAD

**Issue by FagnerMartinsBrack** [js-cookie/js-cookie#423]
Probably test in a browser that supports ES6 without using any library and document accordingly.

**Comment by kitak** [comment]
@FagnerMartinsBrack 

I tried the contents of https://github.com/js-cookie/js-cookie/issues/426 , but it seems that it will not be solved by `import * as Cookies from './js-cookie-master/src/js.cookie.js';`.
The Cookies variable is Module object.

<img width="564" alt="2018-04-18 16 20 01" src="https://user-images.githubusercontent.com/228410/38917022-676b9eb0-4324-11e8-8640-ee62006c6c1d.png">

If object's name is not `Cookies`, I can refer to` window.Cookies` and it will work.
I think that this project needs another file that is ESModule target. I'll create p-r.

**Comment by FagnerMartinsBrack** [comment]
It seems documenting ES6 is more trouble than it's worth. I reverted the commit. Let's use this issue to test on browsers without extra dependencies and document accordingly, without assuming that the developer is using another third-party tool.

Revert commit: https://github.com/js-cookie/js-cookie/commit/d8b016be0f3c068ee4be8a121313de90502da2e0

**Comment by rapeflower** [comment]
TypeError: p.a.setCookie is not a function(…)

**Comment by DoubleCouponDay** [comment]
![image](https://user-images.githubusercontent.com/7555725/40595222-b51a94ec-6276-11e8-8a5d-01ad471fe653.png)

Clearly not the case. js.cookie.js is a self invoking closure which adds the module "Cookie" to the global namespace. 

>It seems documenting ES6 is more trouble than it's worth.

Fine but dont mislead your users by claiming this is an AMD module / commonjs module.

**Comment by FagnerMartinsBrack** [comment]
@DoubleCouponDay This thread is about ES6 module, which was reverted from the README, not AMD or CommonJS.

We support Asynchronous Module Loaders (like require.js) and CommonJS specification, see the tests for Node.js [here](https://github.com/js-cookie/js-cookie/blob/8b311b9dab5751fab3b09b58d215262e908e92e1/test/node.js) and require.js [here](https://github.com/js-cookie/js-cookie/blob/8b311b9dab5751fab3b09b58d215262e908e92e1/test/amd.js).

The sentence says "JavaScript Cookie can also be loaded as an AMD or CommonJS module". What's misleading there?

**Comment by DoubleCouponDay** [comment]
My mistake. I attempted to load js-cookie as a module in the past but ran into issues trying to use es6 module loading without a dependency loader. Now that I'm set up with stealjs without issues, I can do stuff like this it seems:

![image](https://user-images.githubusercontent.com/7555725/40632272-5ed00b1e-6328-11e8-95d8-031e542b0692.png)

all the best,
dcd

**Comment by xyzdata** [comment]
# demos

> ??? testing ...

```js

// ES6
import * as Cookies from"js.cookie.js";

// ES5
var Cookies = require("js.cookie.js");

```

**Comment by xgqfrms** [comment]
# cookie getter

> a more simple cookie getter method!

https://www.w3schools.com/js/js_cookies.asp

```js

function getCookie(cname) {
    var name = cname + "=";
    var decodedCookie = decodeURIComponent(document.cookie);
    var ca = decodedCookie.split(';');
    for(var i = 0; i <ca.length; i++) {
        var c = ca[i];
        while (c.charAt(0) == ' ') {
            c = c.substring(1);
        }
        if (c.indexOf(name) == 0) {
            return c.substring(name.length, c.length);
        }
    }
    return "";
}

```

**Comment by carhartl** [comment]
Closed via #537
